console/src/telemetryFilter.ts

```diff
@@ -10,14 +10,14 @@ const filterEventsBlockList = [
   'RNS_REMOVE_ALL_NOTIFICATIONS',
 ];
 
-const filterPayloadAllowList = [];
+const filterPayloadAllowList: unknown[] = [];
```

Comment:
It's not used anywhere, let's remove it.

Reply:
When I remove the type of `unknown[]`, I get error `Variable 'filterPayloadAllowList' implicitly has type 'any[]' in some locations where its type cannot be determined.` So instead of removing and having the type of `any`, it's better to keep `unknown` there. What do you think?

Reply:
I meant to remove the `filterPayloadAllowList` at all since it's not used anywhere outside of this file.

Reply:
And it's only declared and exported in `telemetryFilter.ts`.

Reply:
great, I do it!